Added Remove Subject option in Grade Calculator

dadigavasavipriya opened the pull request:
### Fixes Issue🛠️ 

Closes #1002 

### Description👨‍💻
Added Remove Subject option in Absolute Grade Calculator and changed marks input format 

### Type of change📄
- [x] New feature (non-breaking change which adds functionality)

### How this has been tested✅
I have tested in my own local host.

### Checklist✅
 
- [x] My code follows the style guidelines of this project
- [x]  I have performed a self-review of my own code
- [ ]  I have commented my code, particularly in hard-to-understand areas
- [x] I have added demonstration in the form of GIF/video file
- [x]  I am an Open Source Contributor

### Screenshots/GIF📷

https://github.com/Rakesh9100/CalcDiverse/assets/112770492/c7cb22b7-4733-4950-b3a6-7512308bb25e

Rakesh9100 commented:
Pls mention the issue number of this PR and also fill every details in the PR template properly @dadigavasavipriya

dadigavasavipriya commented:
### Fixes Issue🛠️ 
Closes#1002

### Description👨‍💻
Added Remove Subject option in Absolute Grade Calculator and changed marks input format 

### Type of change📄

- [ ] New feature (non-breaking change which adds functionality)

dadigavasavipriya commented:
@Rakesh9100 please review the details in PR template.please let me know any changes
